asreview/models/classifiers/lstm_base.py

```diff
@@ -86,6 +86,7 @@ class LSTMBaseClassifier(BaseTrainClassifier):
     """
 
     name = "lstm-base"
+    label = "LSTM classic"
```

Comment:
what is the difference between `base` and `classic`? If there is none, I would not suggest to use both terms. Maybe a more informative label can be provided? `LSTM layer with one output, a dense layer, and a single sigmoid output node`